lib/config.js

```diff
@@ -333,7 +333,11 @@ function Config(options) {
 
     if (useConfig) {
         debug("Using command line config " + useConfig);
-        this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
+        try {
+            this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
```

Comment:
Sorry, I wasn't very clear. I meant we should behave like `require` but not actually use `require`. So does must begin with a non-alphanumeric character and otherwise we assume it's a package.